### PR TITLE
Add operator * for multiplying FloatTransform2D and 3D with each other.

### PR DESCRIPTION
**Pull request by bettinaselig**
@marcusnaslund Peer review?

### CODE REVIEW

**Comment by marcusnaslund** [comment]
I think you can just write `new` without the `~` suffix?

File: source/geometry/FloatTransform3D.ooc [hunk 1]
```diff
@@ -205,6 +205,10 @@ FloatTransform3D: cover {
 			this c * other x + this g * other y + this k * other z + this o
 		)
 	}
+	operator * (other: FloatTransform2D) -> This {
+		otherTransform3D := this new~fromFloatTransform2D(other)
```
Comment (marcusnaslund) [inline source/geometry/FloatTransform3D.ooc]: Also, it should be `This` not `this`.

**Comment by bettinaselig** [comment]
Updated.

**Comment by marcusnaslund** [comment]
Looks good. You should probably make the pull request towards `secondaryFeature1.5`, though.

**Pull request merged**